### PR TITLE
add JWT utilities

### PR DESCRIPTION
**Pull request by HGChamara**
Merging initial JWT utilities to the master

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1 @@
-
+jwt.secret=123456789
```
Comment (putchar91) [inline src/main/resources/application.properties]: Password is not secure

File: src/main/java/com/putchar/jwtgenvalidate/controller/LandingController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.putchar.jwtgenvalidate.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping({"/api"})
```
Comment (putchar91) [inline src/main/java/com/putchar/jwtgenvalidate/controller/LandingController.java]: this is too general

**Review by putchar91 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (HGChamara) [reply src/main/resources/application.properties]: changed the password

File: src/main/java/com/putchar/jwtgenvalidate/controller/LandingController.java (hunk 2 above)
Comment (HGChamara) [reply src/main/java/com/putchar/jwtgenvalidate/controller/LandingController.java]: change api to landing_api

**Comment by HGChamara** [comment]
closing

**Pull request closed without merging**